ISSUE_TEMPLATE

```diff
@@ -0,0 +1,17 @@
+## Expected  behavior
+
+
+## Actual behavior
+
+
+## Steps to reproduce
```

Comment:
Really? Most of the templates I saw in life have this form. But, I'm fine either way of course. :)